### DISCUSSION TITLE
A2DP Source - sample rate different from 44100 Hz

### DISCUSSION THREAD

**Discussion by MiAleKro** [pschatzmann/ESP32-A2DP#235] [Q&A] [answered]
Is there a possibility to run a2dp source with sample rate different from standard 44100 Hz?

**Comment by pschatzmann** [comment, marked as answer]
To my knowledge this is not possible and hard coded into the compiled ESP32 Arduino / PlatformIO code. 

I think in ADF you could try to change the implementation in btc_a2dp_source.c
